manager/src/main/java/com/redhat/service/bridge/manager/providers/InternalKafkaConfigurationProvider.java

```diff
@@ -8,6 +8,4 @@
     String getBootstrapServers();
 
     String getSecurityProtocol();
-
-    String buildTopicName(String bridgeId);
```

Comment:
@kostola It feels like this change removes the single-responsibility principle we have over how Kafka configuration for a Bridge instance is generated. It's now partly done by our implementation of this class and also partly done in the `Bridge` entity.

Was there a reason you decided to split it out?

Reply:
I refactored this part and brought this part back to this class :+1: